README.md

```diff
@@ -116,8 +163,34 @@ Suppose the '&' key on your keyboard is missing, and you want to be able to cont
 **a)**
 Write a function `and2(a,b)` which tries to simulate the && operator: it should always return the same result as `(a && b)` for any values of _a_ and _b_.  (For example, `and2((0>1),true)` should return _false_.)  But you can't use && itself within your function!
 
+function and2(a,b) {
+if (a == true) {
+if (b == true) {
+return 'true';
+}; 
+} else {
+return 'false';
+};
+};
+
```

Comment:
You're definitely on the right track here! Here's a simplified version of what you were going for!

function and2(a,b) {
	if (a) {
		return b;
	} else {
		return a;
	}
}